crypto/include/aes_gcm.h

```diff
@@ -59,4 +62,26 @@ typedef struct {
     srtp_cipher_direction_t dir;
 } srtp_aes_gcm_ctx_t;
 
-#endif /* AES_GCM_OSSL_H */
+#endif /* OPENSSL */
+
+#ifdef NSS
+
+#include <pk11pub.h>
+
+#define MAX_AD_SIZE 2048
+
+typedef struct {
+    int key_size;
+    int tag_size;
+    srtp_cipher_direction_t dir;
+    uint8_t key[32];
+    uint8_t iv[12];
```

Comment:
I'm not up with the use of PK11Context, but wouldn't that be better?

Reply:
I think I tried this and found that PK11Context can't be used with AEAD algorithms.